en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md

```diff
@@ -63,58 +39,47 @@ Follow these steps to set up the ESB Solution Project and the Connector Exporter
             </inSequence>
             <outSequence/>
             <faultSequence/>
-        </target>
-    </proxy>
+           </resource>
+    </api>
     ```
-2. Right-click on the Composite Application Project and click on **Export Project Artifacts and Run**. Select **Run on Micro Integrator**.
+Now we can export the imported connector and the API into a single CAR application. CAR application is the one we are going to deploy to server runtime. 
 
-3. Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
-
-    ```
-    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
-    ```
-
-{! /references/connectors/exporting-artifacts.md !}
+{!/references/connectors/exporting-artifacts.md!}
 
 ## Deployment
+
 Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
 
 {!references/connectors/deploy-capp.md!}
     
 ## Testing
 
-1. Login in to the Micro Integrator CLI tool.
+   **Create a topic**:
```

Comment:
Why have we tabbed this? Tab can be removed for this section.

Reply:
Tabs removed.